go/vt/discovery/healthcheck.go

```diff
@@ -120,12 +120,11 @@ const (
 )
 
 func init() {
-	loadTabletURLTemplate()
+	LoadHealthCheckTabletURLTemplate()
```

Comment:
This should be removed then?

Instead, add a comment to the function "LoadHealthCheckTabletURLTemplate" that it should only be called after the flags are parsed?

Reply:
I left it intentionally because there was one template test that was failing without it. Fixing the test is straightforward, but I was worried that there might be a case out in the while where the template is used before  `LoadHealthCheckTabletURLTemplate` is called. To be safe, I decided to leave the init. 

Are you confident that is safe to remove?

Reply:
Okay, scratch what I said earlier. I understand the problem now.

I suggest the following:

- Rename the method to `ParseTabletURLTemplateFromFlag`.
- Add here a comment: "Flags are not parsed at this point and the default value of the flag (just the hostname) will be used."
- And a comment in the other file: "Flags are parsed now. Parse the template using the actual flag value and overwrite the current template."

What do you think?

Reply:
I think that makes. It will be clearer this way.